js_modules/dagit/packages/core/src/nav/FlatContentList.tsx

```diff
@@ -80,8 +81,13 @@ export const FlatContentList: React.FC<Props> = (props) => {
           <JobItem
             key={`${job.name}-${repoAddressAsString(job.repoAddress)}`}
             job={job}
-            repoPath={repoPath}
-            selector={selector}
+            active={
+              !!(
+                repoPath &&
+                job.repoAddress === repoAddressFromPath(repoPath) &&
```

Comment:
Eeee I'm not wild about relying on the memoization function returning the same instance of the repoAddress both times the function is called (here and in buildJobItems). Since we're already converting the job's repoAddress to a string above I'd much rather convert repoPath to a string before rendering and then comparing the two strings for equality. 

I feel like comparing the URL of the job to the URL of the current page might also work and be a bit more straightforward since they're strings already?